pkg/sfu/buffer/rtpstats.go

```diff
@@ -1187,6 +1187,10 @@ func (r *RTPStats) getAndResetSnapshot(snapshotId uint32) (*Snapshot, *Snapshot)
 // ----------------------------------
 
 func AggregateRTPStats(statsList []*livekit.RTPStats) *livekit.RTPStats {
+	if len(statsList) == 0 {
```

Comment:
would users of this function deal ok with nil values?

Reply:
Will check on that.

Reply:
Looks good from the users side.